### PR TITLE
Added examples for clamp_max, max_over_time, sum_over_time functions.

### PR DESCRIPTION
**Pull request by HemantNegi**
Added examples for below functions:
- clamp_max
- max_over_time
- sum_over_time

### CODE REVIEW

File: docs/querying/examples.md [hunk 1]
```diff
@@ -95,3 +95,25 @@ Assuming this metric contains one time series per running instance, you could
 count the number of running instances per application like this:
 
     count by (app) (instance_cpu_time_ns)
+  
+To find out the maximum value of a metric over a period of time, for eg. last 
+6 hours, you can query like this:
+
+    max_over_time(instance_cpu_time_ns[6h])
```
Comment (brian-brazil) [inline docs/querying/examples.md]: This is not good query, the metric isn't following bestpractices for naming and it looks like a counter.

File: docs/querying/examples.md [hunk 2]
```diff
@@ -95,3 +95,25 @@ Assuming this metric contains one time series per running instance, you could
...
+clamp them to the threshold value like this:
+
+    clamp_max(instance_cpu_time_ns, 100)
+    
+You can calculate the rolling sum of value of a metric over a period of time, 
+say 6 hours like below:
+
+    sum_over_time(some_mertic_name[6h])
+    
+Lets assume we have a deduped metric `processing_time_ns` denoting the time
```
Comment (brian-brazil) [inline docs/querying/examples.md]: This isn't what this query does (plus it's a syntax error), and this sort of thing requires much more discussion than makes sense on a page like this.

File: docs/querying/examples.md (hunk 1 above)
Comment (HemantNegi) [reply docs/querying/examples.md]: I don't understand. its the same metric used in other examples also. 

> it looks like a counter
Why do you think so?

File: docs/querying/examples.md (hunk 2 above)
Comment (HemantNegi) [reply docs/querying/examples.md]: I fixed the syntax error. thanks for pointing that out.
Yeah, I know this is a complex example. But this query counts the number of 1's within the given time period.

File: docs/querying/examples.md (hunk 1 above)
Comment (brian-brazil) [reply docs/querying/examples.md]: Hmm, we should fix those examples then. This sort of data is usually exposed as a counter - which the other examples show.

File: docs/querying/examples.md (hunk 2 above)
Comment (brian-brazil) [reply docs/querying/examples.md]: There's more to this than what's here, it's far too complex to include here. It's a blog post in itself.

**Comment by brian-brazil** [comment]
We looked at this in the bug scrub, and the comments above have not been addressed. We're going to close this for now, but if you would like to add additional better examples to this page feel free to send in a PR.

**Pull request closed without merging**